cmake/concepts_ts.cmake

```diff
@@ -0,0 +1,54 @@
+#
+#  Copyright Christopher Di Bella
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+include(FindPackageHandleStandardArgs)
+include(CheckCXXCompilerFlag)
+include(CMakePushCheckState)
+
+message(STATUS "Checking for concept support")
```

Comment:
I think those messages are redundant because `check_cxx_source_compiles` and `check_cxx_compiler_flag` produce a message. I'd double check that though.